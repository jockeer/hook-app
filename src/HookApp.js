import React from 'react'
import PropTypes from 'prop-types'

const HookApp = props => {
  return (
    <h1>Hola mundo</h1>
  )
}

HookApp.propTypes = {}

export default HookApp