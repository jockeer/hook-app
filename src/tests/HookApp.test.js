import React from 'react';

import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import HookApp from '../HookApp'

describe('Probando componente <HookApp />', () => { 

    test('Debe mostrarse el componente correctamente', () => { 

        const wrapper = shallow(<HookApp />)
        expect( wrapper ).toMatchSnapshot();

    })

})