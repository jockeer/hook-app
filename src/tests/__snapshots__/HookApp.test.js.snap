// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando componente <HookApp /> Debe mostrarse el componente correctamente 1`] = `
<h1>
  Hola mundo
</h1>
`;
